scripts/src/actions/add-gui-workflows.ts

```diff
@@ -0,0 +1,41 @@
+import 'reflect-metadata'
+import * as terraform from '../terraform'
+import * as yaml from '../yaml'
+import YAML from 'yaml'
+
+// TODO: run this script as part of some workflow (PR?, sync?), for now run:
+//       npm run build && TF_WORKSPACE=ipfs node lib/actions/add-gui-workflows.js
+async function run(): Promise<void> {
+  const config = yaml.getConfig()
+
+  // TODO: we need a better abstraction for things that are defined in the YAML
+  //       ideally, we should be able to replace all of this with something like:
+  //       config.getRepositories().filter(repository => repository.topics.includes('ipfs-desktop')).map(repository => repository.name)
+  const repositories = config.getResources([terraform.GithubRepository]).filter(repository => {
+    return ((repository.value as YAML.Pair).value as YAML.YAMLMap).toJSON().topics?.includes('ipfs-desktop')
+  }).map(repository => {
+    return (repository.value as YAML.Pair).key as string
+  })
+
+  for (const repository of repositories) {
+    // TODO: we need a better abstraction here, one which has a proper constructor
+    //       and doesn't know anything about terraform (i.e. values.id)
+    const file = new terraform.GithubRepositoryFile()
+    file.values = {
+      id: '',
+      file: '.github/workflows/auto-project.yml',
```

Comment:
This is the filename as it would exist in the GUI repo, right? We should probably keep the names the same, i.e. `add_issues_to_gui_project.yml`

Reply:
Good call, I'll take care of the rename when I'm doing the rebase.